src/views/juttle-view.js

```diff
@@ -77,19 +77,15 @@ var JuttleView = Base.extend({
         }
     },
 
-    _consume_mark: function(time) {}, //to be overridden as part of the consume_mark template pattern defined below
-    consume_mark: function(time) {
-        if (_.isString(time)) {
-            time = new Date(time);
-        }
-
+    _consume_mark: function(mark) {}, //to be overridden as part of the consume_mark template pattern defined below
+    consume_mark: function(mark) {
         if (this._paused) {
-            this.pauseQueue.push(this.consume_mark.bind(this, time));
+            this.pauseQueue.push(this.consume_mark.bind(this, mark));
             return;
         }
 
         if (_.isFunction(this._consume_mark)) {
```

Comment:
Do we need this `isFunction` check given we have a default function implementation above?